src/tests/configure_nerve_test.py

```diff
@@ -208,6 +209,12 @@ def get_named_zookeeper_topology(cluster_type, cluster_location, zk_topology_dir
             zk_topology_dir='/fake/path',
             zk_location_type='superregion',
             zk_cluster_type='infrastructure',
+            location_depth_mapping={
```

Comment:
Maybe generate the mock data with the same logic we use for "realz" (maybe move the depth mapping calculation into a function or something).